jcl/src/jdk.jlink/share/classes/jdk/tools/jlink/internal/plugins/GenerateJLIClassesPlugin.java

```diff
@@ -19,4 +19,22 @@
  * [2] http://openjdk.java.net/legal/assembly-exception.html
  *
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
- *******************************************************************************/
\ No newline at end of file
+ *******************************************************************************/
+package jdk.tools.jlink.internal.plugins;
+
+import jdk.tools.jlink.plugin.Plugin;
+import jdk.tools.jlink.plugin.ResourcePool;
+import jdk.tools.jlink.plugin.ResourcePoolBuilder;
+import jdk.tools.jlink.plugin.ResourcePoolEntry;
+import java.util.function.Function;
+
+public class GenerateJLIClassesPlugin implements Plugin
+{
+   public ResourcePool transform( ResourcePool rp, ResourcePoolBuilder rpb ){
+      Function<ResourcePoolEntry, ResourcePoolEntry> returnsArg = (val) -> {
+         return val;
+      };
+      rp.transformAndCopy(returnsArg,rpb);
+      return rpb.build();
+   }
+}
```

Comment:
Can you add a newline at the end of the file?

Reply:
Done.